src/Adapters/Laravel/Commands/TestCommand.php

```diff
@@ -77,7 +77,7 @@ public function handle()
                 "\n  <fg=white;bg=red;options=bold> ERROR </> Code coverage driver not available.%s</>",
                 Coverage::usingXdebug()
                     ? " Did you set <href=https://xdebug.org/docs/code_coverage#mode>Xdebug's coverage mode</>?"
-                    : ''
+                    : " Did you install <href=https://xdebug.org/>Xdebug</> or <href=https://pecl.php.net/package/pcov>PCOV</>?"
```

Comment:
Maybe point people towards the GitHub repo? 🤔
```suggestion
                    : " Did you install <href=https://xdebug.org/>Xdebug</> or <href=https://github.com/krakjoe/pcov>PCOV</>?"
```

Reply:
I used these links: https://laravel.com/docs/9.x/testing#reporting-test-coverage. No problem for me using the repository. I just found another line. PR will be updated in a few.